sql/core/src/test/resources/tpcds-query-results/v1_4/q17.sql.out

```diff
@@ -3,4 +3,4 @@
 -- !query schema
 struct<i_item_id:string,i_item_desc:string,s_state:string,store_sales_quantitycount:bigint,store_sales_quantityave:double,store_sales_quantitystdev:double,store_sales_quantitycov:double,as_store_returns_quantitycount:bigint,as_store_returns_quantityave:double,as_store_returns_quantitystdev:double,store_returns_quantitycov:double,catalog_sales_quantitycount:bigint,catalog_sales_quantityave:double,catalog_sales_quantitystdev:double,catalog_sales_quantitycov:double>
 -- !query output
-AAAAAAAAKPFEAAAA	Recently right	TN	1	99.0	NULL	NULL	1	66.0	NULL	NULL	1	32.0	NULL	NULL
```

Comment:
I've checked that the result is the same when using StringType instead of CharType. It looks `sf=1` is not enough to output valid rows, e.g., if sf=20, this result will be;
```
+AAAAAAAAAAMHBAAA       English, labour shares put to a sanctions. Central, following opinions will wreck workers; curious, real qualities could not involve nearly most national parties. Unable, detailed services    SD      1       49.0    NULL    NULL    1       8.0     NULL    NULL    1       68.0    NULL    NULL
+AAAAAAAAGBKDAAAA       Other men can keep in a customers. Surprised premises might not allow. Technical, british cler  SD      1       68.0    NULL    NULL    1       11.0    NULL    NULL    1       30.0    NULL    NULL
+AAAAAAAAGEOBBAAA       Safe, rough minutes shall find again famous orders. British     SD      1       38.0    NULL    NULL    1       22.0    NULL    NULL    1       88.0    NULL    NULL
+AAAAAAAAIBLFBAAA       Communications love with a texts. Powerful proceedings should feel players. Full names should not cover various, full spaces. Clubs must intervene in the ho    AL      1       2.0     NULL    NULL    1       2.0     NULL    NULL    1       49.0    NULL    NULL
+AAAAAAAAKOJIAAAA       Areas inform empirical scientists. Old  TN      1       68.0    NULL    NULL    1       21.0    NULL    NULL    1       87.0    NULL    NULL
+AAAAAAAAKPMNAAAA       Generally great holidays must keep separately very domestic cases; doctors explain then nuclear friends. Systemat       AL      1       4.0     NULL    NULL    1       4.0     NULL    NULL    1       64.0    NULL    NULL
+AAAAAAAAONIMAAAA       Tempera AL      1       58.0    NULL    NULL    1       14.0    NULL    NULL    1       73.0    NULL    NULL
+AAAAAAAAOPAJAAAA       Scots will not hang children. Long groups should not need also ol       TN      1       59.0    NULL    NULL    1       9.0     NULL    NULL    1       32.0    NULL    NULL
```